tests/psi4numpy/rhf_gradient/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+include(TestingMacros)
+
+add_regression_test(psi4numpy-rhf_gradient "psi;quicktests;smoketests;psi4numpy")
```

Comment:
tests use dash rather than underscore in their names, so that'll need altering in several places.

Reply:
Done.